app/models/organization.rb

```diff
@@ -357,7 +359,7 @@ def setup_available_statuses
   def get_available_statuses
     tmp_available_statuses = self.available_statuses.reject{|status| status.new_record?}
     statuses = []
-    if tmp_available_statuses.empty?
+    if tmp_available_statuses.empty? || tmp_available_statuses.collect{|status| status.status} == DEFAULT_STATUSES.collect{|k, v| k}
```

Comment:
It may be shorter to do `tmp_available_statuses.map(&:status) == DEFAULT_STATUSES.keys`